src/Microsoft.TestPlatform.CrossPlatEngine/AttachmentsProcessing/TestRunAttachmentsProcessingManager.cs

```diff
@@ -91,52 +92,87 @@ private async Task<Collection<AttachmentSet>> InternalProcessTestRunAttachmentsA
             {
                 EqtTrace.Error("TestRunAttachmentsProcessingManager: Exception in ProcessTestRunAttachmentsAsync: " + e);
 
-                eventHandler?.HandleLogMessage(TestMessageLevel.Error, e.Message);
+                eventHandler?.HandleLogMessage(TestMessageLevel.Error, "TestRunAttachmentsProcessingManager: Exception in ProcessTestRunAttachmentsAsync: " + e);
```

Comment:
Moved back to `e.Message` and aligned the other exception too so we can avoid translation here, we have the logs above also in case of issue and the message of exception will be already translated.